content/docs/dvclive/api-reference/live/index.md

```diff
@@ -9,6 +9,8 @@ class Live:
         self,
         path: Optional[str] = None,
         resume: bool = False,
+        report: Optional[str] = "html",
+        auto_open: bool = True,
```

Comment:
I think this should be `False` according to https://github.com/iterative/dvclive/pull/240?